meta/documents/user_guide_de.md

```diff
@@ -1,253 +1,49 @@
-
-Die Einstellungen sollten am Ende so aussehen:
-![Add Procedure Result](https://i.ibb.co/GJPF3ZV/asn-06.png "Add Procedure Result")
+Um sicherzustellen, dass Sie die neueste Version dieses Dokuments sehen, [klicken Sie hier](https://github.com/wayfair-contribs/plentymarkets-plugin/blob/main/meta/documents/user_guide_de.md).
+
+## Willkommen bei Wayfair
+​
+### Einführung in das Wayfair-Plugin
+Das Wayfair-Plugin ist das offizielle Plugin für die Verwendung von Plentymarkets zum Verkauf von Artikeln auf dem Wayfair-Marktplatz.
+Das Plugin bietet mehrere Integrationen in die Schnittstellen von Wayfair, einschließlich - aber nicht beschränkt auf - die folgenden Vorgänge:
+
+* Automatischer Abruf von Wayfair-Bestellungen mit Umstellung auf Plentymarkets-Bestellungen
+​
+* Automatisches Senden von Plentymarkets-Inventarinformationen an Wayfair, um die Artikelverfügbarkeit genau zu halten
+​
+* Automatische Übertragung von Wayfair-generierten Versanddokumenten an Plentymarkets
+​
+* Änderungen des Bestellstatus von Plentymarkets an Wayfair senden, um Wayfair-Kunden und den Wayfair-Kundensupport auf dem neuesten Stand zu halten.
+
+### Verkauf auf Wayfair
+Wayfair ist ein geschlossener Marktplatz. Die Verwendung der Wayfair Plugin ist auf Organisationen beschränkt, die sich als Wayfair-Lieferanten registriert haben.
+​
+Wenn Sie derzeit nicht auf Wayfair verkaufen, gehen Sie bitte zu https://partners.wayfair.com/d/onboarding/sell-on-wayfair, um loszulegen.
+
+### Anweisungen für aktuelle Wayfair-Lieferanten
+Wenn Sie ein Wayfair-Lieferant sind, der das Wayfair Plentymarkets-Plugin verwenden möchte, lesen Sie Bitte lesen Sie diese Bedienungsanleitung um zu erfahren, wie Sie die richtigen Vereinbarungen mit Wayfair treffen.
+​
+## Vorbereitung zur Verwendung des Wayfair-Plugins
+Für die Verwendung des Wayfair-Plugins sind Wayfair-API-Anmeldeinformationen erforderlich, denen die entsprechenden Rechte erteilt wurden.
+
+Verwenden Sie vor der Installation [diese Anweisungen](https://github.com/wayfair-contribs/plentymarkets-plugin/blob/main/meta/documents/user_guide/de/obtaining_credentials.md), um Wayfair zu kontaktieren und sich einzurichten.
+​
+## Installieren des Wayfair-Plugins
+Das Wayfair-Plugin ist kostenlos auf dem PlentyMarketplace erhältlich. [Klicken Sie hier](https://marketplace.plentymarkets.com/de/plugins/integration/wayfair_6273), um die Seite des Wayfair Plugins anzuzeigen.
+
+Installieren Sie das Plugin im aktiven Plugin-Set in Plentymarkets. Anleitungen von Wayfair finden Sie [hier](https://github.com/wayfair-contribs/plentymarkets-plugin/blob/main/meta/documents/user_guide/de/plugin_installation.md).
+
+## Einrichten des Wayfair-Plugins
+Bevor Sie das Plugin zum ersten Mal verwenden, [verwenden Sie diese Anweisungen](https://github.com/wayfair-contribs/plentymarkets-plugin/blob/main/meta/documents/user_guide/de/initial_setup.md),  um das Plugin mit Wayfair zu verbinden und dessen Funktionalität sicher zu stellen.
```

Comment:
**Broken URL:** URL points to the main branch

Reply:
Discuss offline